workspace/apps/pidp/src/app/features/auth/pages/login/login.page.ts

```diff
@@ -20,13 +28,16 @@ export class LoginPage {
   public providerIdentitySupportEmail: string;
   public specialAuthoritySupportEmail: string;
   public idpHint: IdentityProvider;
+  public collectionNotice: string;
```

Comment:
Was added since the collection notice was consumed in the template, but you could now directly access it via the document service since it remains in the controller and is only used in the dialog.